src/Mod/TechDraw/Gui/Command.cpp

```diff
@@ -1159,7 +1159,7 @@ void CmdTechDrawDraftView::activated(int iMsg)
     }
     if (draftItemsFound == 0) { 
         QMessageBox::warning(Gui::getMainWindow(), QObject::tr("Wrong selection"),
-            QObject::tr("There were no DraftWB objects in the selection."));
+            QObject::tr("There were no DraftWB objects in the selection.\n\nIf you wish to group Draft objects then use Layers in order for them to be collected."));
```

Comment:
Please find a way to not include the newlines in the translation. I'm not sure what needs to happen? Does a new QObject need to be created to do so?

Reply:
Are the newlines needed in this case? I think it would work without.

Reply:
Probably not.